build/docker/tools/provider-res/Dockerfile

```diff
@@ -26,14 +26,26 @@ COPY linux-arm64/cni-*.tar.gz          res/linux-arm64/
 COPY linux-amd64/conntrack-tools-*.tar.gz          res/linux-amd64/
 COPY linux-arm64/conntrack-tools-*.tar.gz          res/linux-arm64/
 
+COPY linux-amd64/nerdctl-*.tar.gz res/linux-amd64/
+COPY linux-arm64/nerdctl-*.tar.gz res/linux-arm64/
+
 COPY linux-amd64/docker-*.tar.gz       res/linux-amd64/
 COPY linux-arm64/docker-*.tar.gz       res/linux-arm64/
 
+COPY linux-amd64/containerd-*.tar.gz       res/linux-amd64/
+COPY linux-arm64/containerd-*.tar.gz       res/linux-arm64/
+
+COPY linux-amd64/critools-*.tar.gz       res/linux-amd64/
+COPY linux-arm64/critools-*.tar.gz       res/linux-arm64/
```

Comment:
why need package critools? can you rebuild a containerd package which include critools and push it to cos ?

Reply:
Remove it.